presto-main/src/main/java/com/facebook/presto/metadata/FunctionAndTypeManager.java

```diff
@@ -274,7 +274,7 @@ public List<SqlFunction> listFunctions(Session session, Optional<String> likePat
                 .collect(toImmutableList());
     }
 
-    public Collection<SqlFunction> listBuiltInFunctions()
+    public List<SqlFunction> listBuiltInFunctions()
```

Comment:
Needs javadoc specifying the return value is immutable since that is *not* included in the type signature.